contest/views.py

```diff
@@ -222,3 +222,11 @@ def reply(request):
                     % (request.user.username, replied_clarification.id))
             return redirect('contest:contest',contest)
     return redirect('contest:archive')
+
+def download(request):
+    what = request.POST.get('type')
+    if what == 'scoreboard':
```

Comment:
Maybe ``request.method == 'POST' and 'scoreboard' in request.POST`` would be better.
Btw, does download the file require any permission?